src/renderer/components/dialogs/ForwardMessage.js

```diff
@@ -7,7 +7,7 @@ const {
 
 const ContactList = require('../ContactList')
 
-class ForwardMessageDialog extends React.Component {
+class ForwardMessage extends React.Component {
```

Comment:
Now all Dialogs need to have their filename match the component they're exporting so that the controller doesn't get confused